cartography/models/aws/identitycenter/permissionsets.py

```diff
@@ -0,0 +1,44 @@
+from dataclasses import dataclass
+
+from cartography.models.core.common import PropertyRef
+from cartography.models.core.nodes import CartographyNodeProperties
+from cartography.models.core.nodes import CartographyNodeSchema
+from cartography.models.core.relationships import CartographyRelProperties
+from cartography.models.core.relationships import CartographyRelSchema
+from cartography.models.core.relationships import LinkDirection
+from cartography.models.core.relationships import make_target_node_matcher
+from cartography.models.core.relationships import TargetNodeMatcher
+
+
+@dataclass(frozen=True)
+class PermissionSetProperties(CartographyNodeProperties):
+    id: PropertyRef = PropertyRef('PermissionSetArn')
+    name: PropertyRef = PropertyRef('Name')
+    arn: PropertyRef = PropertyRef('PermissionSetArn')
+    description: PropertyRef = PropertyRef('Description')
+    session_duration: PropertyRef = PropertyRef('SessionDuration')
+    instance_arn: PropertyRef = PropertyRef('InstanceArn', set_in_kwargs=True)
+    lastupdated: PropertyRef = PropertyRef('lastupdated', set_in_kwargs=True)
+
+
+@dataclass(frozen=True)
+class PermissionSetToInstanceRelProperties(CartographyRelProperties):
+    lastupdated: PropertyRef = PropertyRef('lastupdated', set_in_kwargs=True)
+
+
+@dataclass(frozen=True)
+class PermissionSetToInstance(CartographyRelSchema):
+    target_node_label: str = 'AWSIdentityCenter'
+    target_node_matcher: TargetNodeMatcher = make_target_node_matcher(
+        {'arn': PropertyRef('InstanceArn', set_in_kwargs=True)},
+    )
+    direction: LinkDirection = LinkDirection.INWARD
+    rel_label: str = "HAS_PERMISSION_SET"
+    properties: PermissionSetToInstanceRelProperties = PermissionSetToInstanceRelProperties()
+
+
+@dataclass(frozen=True)
+class PermissionSetSchema(CartographyNodeSchema):
+    label: str = 'AWSPermissionSet'
+    properties: PermissionSetProperties = PermissionSetProperties()
+    sub_resource_relationship: PermissionSetToInstance = PermissionSetToInstance()
```

Comment:
sub_resource_relationship is intended to point to the AWSAccount (yeah, it's not a great name, I have an issue open to rename it to something like "tenant_relationship"). It is used for autocleanup jobs so that we only delete stale nodes in the current account.